CHANGELOG.md

```diff
@@ -5,6 +5,108 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.0.0-alpha.14
+
+_Mar 6, 2025_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 🛠️ New and improved Toolbar component for the data grid
```

Comment:
@mui/xcharts do you want to highlight the new charts? 🤔

Reply:
added it @mui/xcharts 👍🏼